packages/block-library/src/list/transforms.js

```diff
@@ -114,17 +114,6 @@ const transforms = {
 				);
 			},
 		} ) ),
-		{
-			type: 'block',
-			blocks: [ '*' ],
-			transform: ( _attributes, childBlocks ) => {
-				return getListContentFlat( childBlocks ).map( ( content ) =>
-					createBlock( 'core/paragraph', {
-						content,
-					} )
-				);
-			},
-		},
```

Comment:
Checking my assumptions here: this bit that we removed — i.e. the _Unwrap_ transform — was functionally equivalent to the Transform to Paragraph transform, right? So we don't lose anything?

Reply:
Yes, it was a hackier way of doing list item to paragraph. It wasn't great because '*' should mean unwrap, while this is unwrapping and converting it to paragraphs.